libr/core/cmd_anal.c

```diff
@@ -1360,18 +1344,20 @@ static int var_cmd(RCore *core, const char *str) {
 			return false;
 		}
 		if (str[2] == '*') {
-			r_anal_var_delete_all (core->anal, fcn->addr, type);
+			r_anal_function_delete_all_vars_of_kind (fcn, type);
```

Comment:
this is not java

Reply:
give me a better name

Reply:
r_anal_function_delete_vars_by_kind() or r_anal_fdvbk()

Reply:
be kind

Reply:
`r_anal_fdvbk()` sgtm

Reply:
this one sounds specially good with german accent

Reply:
r_anal_effdefaubeka()